vassal-app/src/main/java/VASSAL/build/module/GameState.java

```diff
@@ -581,47 +581,65 @@ public boolean isSaveMetaDataValid(File file) {
     // Check it belongs to this module and matches the version if is a
     // post 3.0 save file
     final SaveMetaData saveData = (SaveMetaData) metaData;
+
     final String moduleName = GameModule.getGameModule().getGameName();
     final String moduleVersion = GameModule.getGameModule().getGameVersion();
+    final String vassalVersion = VersionUtils.truncateToIncrementalVersion(Info.getVersion());
+
     final String saveModuleName = saveData.getModuleName();
     String saveModuleVersion = "?";
+    String saveVassalVersion = "?";
+    int problemCount = 0;
+
+    // Was the Module Data that created the save stored in the save? (Vassal 3.0+)
     if (saveData.getModuleData() != null) {
       saveModuleVersion = saveData.getModuleVersion();
-      String message = null;
+      saveVassalVersion = VersionUtils.truncateToIncrementalVersion(saveData.getVassalVersion());
+      final StringBuilder message = new StringBuilder();
+      message.append(Resources.getString("GameState.load_mismatch_header",  file.getName())).append("\n\n");
 
       if (!saveModuleName.equals(moduleName)) {
-        message = Resources.getString(
-          "GameState.load_module_mismatch",
-          file.getName(), saveModuleName, moduleName
-        );
+        message.append(Resources.getString("GameState.load_module_mismatch", ++problemCount, saveModuleName, moduleName)).append('\n');
```

Comment:
I think it looks a bit odd displaying `(1)` when there's only one item. Is there a need to number the items? Could we use a bullet point instead?

Reply:
Agreed. Updated.